style.css

```diff
@@ -0,0 +1,98 @@
+*{
+    margin: 0;
+    padding: 0;
+
+}
+Body{
```

Comment:
this should be `body` lower case

Reply:
oh sorry definitely lower case, thank you.